### DISCUSSION TITLE
'No authority' when trying to activate SolisCloud API

### DISCUSSION THREAD

**Discussion by YanisKyr** [hultenvp/solis-sensor#221] [Q&A] [answered]
I am getting this error when trying to activate the SolisCloud API. Any ideas what's blocking me?

https://i.imgur.com/yTTG9Pi.jpg

**Comment by hultenvp** [comment]
No, I don't. Perhaps if you open a topic in discussion others might be able to help.
Converting to discussion

**Comment by peledudok** [comment]
did you create a request to get an API at Solisloud support?

**Reply by YanisKyr** [reply]
I haven't - I saw the option in the portal and thought "Agree and activate" was all to it.

How do I open a request with their support? There are so many websites in the results...

**Reply by wyliecoyoteuk** [reply]
register here and raise a ticket
https://solis-service.solisinverters.com/support/login

**Reply by peledudok** [reply]
<img width="253" alt="image" src="https://user-images.githubusercontent.com/88967555/204310442-bff61727-19bc-4be3-b145-3837a25bd756.png">

**Reply by YanisKyr** [reply]
Thanks guys, I'll try this.

I'm noticing Remote access is an option too - that seems to enable changing settings via the app according to my searches. Would you know if I can enable both or if they are conflicting?

**Reply by peledudok** [reply]
you mean with remote https://www.soliscloud.com/ ?
i use the that in combination with also the app.
i'v got no clue if that is conflicting.

**Reply by wyliecoyoteuk** [reply]
> Thanks guys, I'll try this.
> 
> I'm noticing Remote access is an option too - that seems to enable changing settings via the app according to my searches. Would you know if I can enable both or if they are conflicting?

Yes I have both working. you will need to raise a separate ticket.

**Comment by YanisKyr** [comment, marked as answer]
Just circling back to this - API access was granted and this solved the problem.
